qiskit/transpiler/passes/scheduling/base_scheduler.py

```diff
@@ -269,6 +269,23 @@ def _get_node_duration(
         else:
             duration = node.op.duration
 
+        if isinstance(node.op, Reset):
+            raise RuntimeWarning(
```

Comment:
raise is invalid syntax 
https://docs.python.org/ja/3/library/warnings.html#warnings.warn

the rest of the PR looks good to me.

Reply:
@nkanazawa1989 Why is this invalid syntax? Are you saying that we should be using warn (UserWarning) instead of RuntimeWarning?

Reply:
You need to import warnings and use `warnings.warn` instead of raise; https://stackoverflow.com/questions/3891804/raise-warning-in-python-without-interrupting-program. Currently linter fails due to this and I cannot merge.